### ISSUE TITLE
datagov-ssb -- 'The request is invalid'

### ISSUE THREAD

**Issue by nickumia-reisys** [GSA/data.gov#3718]
Cloud.gov cli v8
datagov-ssb@55c547dda56568045915a20b198301f021f14e94
solr-brokerpak v0.20.2 AND solr-brokerpak v1.0.7
eks-brokerpak v1.0.2 (only for solr-brokerpak v0.20.2)
eks-brokerpak@another-fix-for-finding-binaries (only for solr-brokerpak v1.0.7)

## How to reproduce

1. Unreliable reproduction..
    - https://github.com/GSA/datagov-ssb/runs/5452640639?check_suite_focus=true#step:5:618
    - https://github.com/GSA/datagov-ssb/runs/5427589503?check_suite_focus=true#step:5:582
    - https://github.com/GSA/datagov-ssb/runs/5427420059?check_suite_focus=true#step:5:592
    - https://github.com/GSA/datagov-ssb/runs/5427076516?check_suite_focus=true#step:5:1303

## Expected behavior

Terraform completes successfully.

## Actual behavior

Terraform fails with 
![image](https://user-images.githubusercontent.com/85196563/157089517-6906a62d-7fad-4f26-88fc-48480e49f396.png)

## Sketch

[Notes or a checklist reflecting our understanding of the selected approach]
- [x] Investigate https://github.com/GSA/datagov-ssb/runs/5426689317?check_suite_focus=true#step:5:566 as it might be related.
- [x] Determine the cause of the invalid request.
  - [ ] (Reach out to [cg-support](https://gsa-tts.slack.com/archives/C09CR1Q9Z))
- [x] Fix the cause of the invalid request.

**Comment by mogul** [comment]
I'm able to reproduce this locally now and will investigate further.

**Comment by mogul** [comment]
I happened across [this PR and the resulting discussion](https://github.com/cloudfoundry-community/terraform-provider-cloudfoundry/issues/273), which points to what was different about the `production` deployment: The target space, `management` has ssh disabled.

I ran `cf allow-space-ssh management`, and then tried applying again. This time I got "The app name is taken"... which is expected because while this was happening, we had redeployed by hand!

I ran 
```bash
terraform import -var-file=terraform.${ENV_NAME}.tfvars 'module.broker_solr.cloudfoundry_app.ssb' SSB_SOLR_APP_GUID
terraform import -var-file=terraform.${ENV_NAME}.tfvars 'module.broker_solrcloud.cloudfoundry_app.ssb' SSB_SOLRCLOUD_APP_GUID
``` 

And Terraform properly took ownership of the apps!

We had to do some other futzing based on having manually set broker passwords last Friday, but, long story short: datagov-ssb terraform is applying cleanly to production again now.
